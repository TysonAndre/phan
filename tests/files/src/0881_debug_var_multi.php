<?php
$var1 = 'foo';
$var2 = 'bar';
$var3 = 42;
'@phan-debug-var $var1, $var2,$var3';
