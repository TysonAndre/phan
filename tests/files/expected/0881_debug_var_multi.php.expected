%s:4 PhanDebugAnnotation @phan-debug-var requested for variable $var1 - it has union type 'foo'(real='foo')
%s:4 PhanDebugAnnotation @phan-debug-var requested for variable $var2 - it has union type 'bar'(real='bar')
%s:4 PhanDebugAnnotation @phan-debug-var requested for variable $var3 - it has union type 42(real=42)
